src/NewFamilyMember.tsx

```diff
@@ -0,0 +1,97 @@
+import { useState } from 'react';
+import './NewFamilyMember.scss';
+
+enum DocumentType {
+  type,
+  dni,
+  nie,
+  pasaporte
```

Comment:
[Typescript enum convention](https://www.typescriptlang.org/docs/handbook/enums.html) recommends PascalCase for enums. I'd also recommend having an explicit value.

Additionally, I understand that you want to use the enum for the values shown, which will be in Spanish, but there are other ways to do that. I would keep the types in english

```suggestion
  type,
  IdCard,
  FIN, // Foreigner Identity Number
  Passport
```